code/modules/clothing/masks/gasmask.dm

```diff
@@ -386,3 +386,17 @@
 	flags_inv = HIDEFACIALHAIR
 	w_class = WEIGHT_CLASS_NORMAL
 	inhand_icon_state = "driscoll_mask"
+
+/obj/item/clothing/mask/gas/prop
+	name = "prop gas mask"
+	desc = "A prop gas mask designed for appearance. Unlike a normal gas mask this does not filter gasses or protect against pepper spray."
+	flags_cover = NONE
```

Comment:
i think you want maskcoversmouth here, weird that people would be able to eat with it